livekit-android-sdk/src/main/java/io/livekit/android/room/Room.kt

```diff
@@ -58,13 +60,24 @@ constructor(
     @Deprecated("Use events instead.")
     var listener: RoomListener? = null
 
-    var sid: Sid? = null
-        private set
-    var name: String? = null
+    /**
+     * Changes can be observed by using [io.livekit.android.util.flow]
+     */
+    var sid: Sid? by flowDelegate(null)
```

Comment:
These flowDelegates are the guts of the compose support, allowing them to be accessed normally like variables, but also observed using the `flow` function to get notified of changes (and can be used in compose for state purposes).